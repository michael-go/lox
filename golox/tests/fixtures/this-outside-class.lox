fun notAMethod() {
  print this;
}

print this;