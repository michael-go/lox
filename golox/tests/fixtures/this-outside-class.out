# exit code: 1
# stdout:

# stderr:
[line 2] Error at 'this': Can't use 'this' outside of a class.
[line 5] Error at 'this': Can't use 'this' outside of a class.
exit status 65

